docs/config/README.md

```diff
@@ -285,6 +286,35 @@ it would have been `nixpkgs/pkgs`.
 truncation_length = 8
 ```
 
+## Environment Variable
+
+The `env_var` module displays the current value of a selected environment variable.
+The module will be shown only if any of the following conditions are met:
+
+- The environment variable named `variable` is defined
+- The environment variable named `variable` is not defined and `default` config value exists
+
+### Options
+
+| Variable   | Default          | Description                                                                  |
+| ---------- | ---------------- | ---------------------------------------------------------------------------- |
+| `symbol`   | `""`             | The symbol used before displaying the variable value.                        |
+| `variable` | `""`             | The environment variable to be displayed.                                    |
+| `default`  | `""`             | The default value to be displayed when the selected variable is not defined. |
```

Comment:
`symbol`, `variable`, `default` don't have a default value. Just leave it blank.

Reply:
This is fixed now